src/pharmpy/model/statements.py

```diff
@@ -1113,7 +1130,10 @@ def _find_depot(self):
         depot = None
         for to_central, _ in self.get_compartment_inflows(central):
             outflows = self.get_compartment_outflows(to_central)
-            if len(outflows) == 1:
+            if len(outflows) == 1 or len(outflows) == 2:
+                if len(outflows) == 2:
+                    if not self.get_flow(to_central, self.find_compartment("METABOLITE")):
```

Comment:
Is this needed? How do we find the central compartment in the presystemic case?

Reply:
Currently, we have no precise way of differentiating the central and metabolite compartments in the presystemic case. So this is currently dependent on the compartment name as previously discussed.